tests/testCoadds.py

```diff
@@ -175,6 +203,14 @@ def comparePsfs(self, a, b):
             for aParam, bParam in zip(aFuncParams, bFuncParams):
                 self.assertEqual(aParam, bParam)
 
+    @unittest.skip("Remove test until DM-5174 is complete")
+    def testMasksRemoved(self):
```

Comment:
Well the code that will need to change is one line that this ticket added and is documented on the ticket. I would rather not loose the work of writing this test, and need to reduplicate it later. Since this is currently not doing anything it should be save to leave in. If in the future the other ticket is marked invalid, it should be clear enough to remove this bit of code too.